### ISSUE TITLE
Add support for providing a URL to monitor

### ISSUE THREAD

**Issue by haydentherapper** [sigstore/rekor-monitor#400]
**Description**

The binary supports a `url` flag, defaulting to the production Rekor instance. The reusable workflow should take an optional parameter for the URL as well, so that the staging instance or private deployments can be monitored.

**Comment by Horiodino** [comment]
@haydentherapper,  let me know if this issue is still open. Is there anything else I should be aware of? Thanks

**Comment by haydentherapper** [comment]
Yep, it is! For this, we just need to add a configurable URL to the two reusable workflows and pass it to the binary with the `--url` flag.
